src/OVAL/public/oval_definitions.h

```diff
@@ -2450,6 +2450,11 @@ void oval_state_content_set_varcheck(struct oval_state_content *, oval_check_t);
  * @memberof oval_state_content
  */
 void oval_state_content_set_entcheck(struct oval_state_content *, oval_check_t);
+/**
```

Comment:
You are right that our doxygen documentation of source code is bad. There are even some functions that do not have any documentation, and most of the functions have only very short description.